src/garage/experiment/local_tf_runner.py

```diff
@@ -193,24 +199,25 @@ def save(self, epoch, paths=None):
         """
         assert self.has_setup
 
-        logger.log('Saving snapshot...')
+        if isinstance(self.snapshot_config, SnapshotConfig):
+            logger.log('Saving snapshot...')
 
-        params = dict()
-        # Save arguments
-        params['setup_args'] = self.setup_args
-        params['train_args'] = self.train_args
+            params = dict()
+            # Save arguments
+            params['setup_args'] = self.setup_args
+            params['train_args'] = self.train_args
 
-        # Save states
-        params['env'] = self.env
-        params['algo'] = self.algo
-        if paths:
-            params['paths'] = paths
-        params['last_epoch'] = epoch
-        snapshotter.save_itr_params(epoch, params)
+            # Save states
+            params['env'] = self.env
+            params['algo'] = self.algo
+            if paths:
+                params['paths'] = paths
+            params['last_epoch'] = epoch
+            self.snapshotter.save_itr_params(epoch, params)
```

Comment:
This line breaks when there's no snapshotter.

Reply:
This will not break as we checked `isinstance(self.snapshot_config, SnapshotConfig)` before. Though I don't know if there is a better way to tell the Runner whether user want to snapshot or not.

Reply:
i don't think it will break because you checked that `if self.snaphotter:`